sites/svelte.dev/src/routes/_components/Demo.svelte

```diff
@@ -98,16 +105,41 @@
 		height: 100%;
 		align-items: center;
 		justify-content: center;
-		border-radius: var(--sk-border-radius) var(--sk-border-radius) 0 0;
+		border-right: 0.5px solid var(--sk-text-4);
```

Comment:
how about lightening up the border a little?
```suggestion
		border-right: 0.5px solid var(--sk-back-4);
```